src/sql/parts/jobManagement/common/media/jobs.css

```diff
@@ -205,4 +205,16 @@ agentview-component .jobview-grid .grid-canvas > .ui-widget-content.slick-row.od
 #jobsDiv jobsview-component .jobview-grid .slick-cell.l1.r1.error-row td.jobview-jobnameindicatorfailure {
     width: 0;
     background: none;
+}
+
+#jobsDiv .jobview-grid > .monaco-table .slick-viewport > .grid-canvas > .ui-widget-content.slick-row:hover > .slick-cell,
+#jobsDiv .jobview-grid > .monaco-table .slick-viewport > .grid-canvas > .ui-widget-content.slick-row.hovered > .slick-cell,
+#jobsDiv .jobview-grid > .monaco-table .slick-viewport > .grid-canvas > .ui-widget-content.slick-row> .slick-cell.hovered {
+    background: #dcdcdc !important;
```

Comment:
we get a lot of accessibility bugs regarding contrast ratio.  could you please check with Abbie that these colors are correct for those guidelines?

Reply:
I got these colors from the tree highlighting specs, so that it's consistent throughout. @AbbiePetcht is this fine?